source/compatibility.tex

```diff
@@ -40,6 +40,20 @@
 \end{codeblock}
 \end{example}
 
+\rSec2[diff.cpp20.expr]{\ref{expr}: expressions}
+
+\diffref{expr.sub}
+\change
+Change the meaning of comma in subscript expressions.
+\rationale
+Enable repurposing a deprecated syntax to support multidimensional indexing.
+\effect
+Valid \CppXX{} code that uses a comma expression within a
+subscript expression may fail to compile. For example:
+\begin{codeblock}
+arr[1, 2]               // was equivalent to \tcode{arr[(1, 2)]}, now equivalent to \tcode{arr.\keyword{operator}[](1, 2)} or ill-formed
```

Comment:
Split the comment into 2 lines after the ","

Reply:
Fixed